recipes/perl-extutils-makemaker/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "Nothing to do."
```

Comment:
Then you shouldn't need this file at all. :)

But, this can't be right: you need to install the files into `PREFIX`, however one does that in perl. As-is, this is making an empty package:
```
2018-02-13 15:15:06,939 - conda_build.build - INFO - Packaging perl-extutils-makemaker-6.66-pl5.22.0.1_0
INFO conda_build.build:bundle_conda(863): Packaging perl-extutils-makemaker-6.66-pl5.22.0.1_0
number of files: 0
```